blockchain-source/src/cardano/mod.rs

```diff
@@ -9,12 +11,11 @@ use cardano_net::{NetworkDescription, NetworkHandle};
 pub use cardano_sdk::protocol::Tip;
 use cardano_sdk::protocol::Version;
 pub use configuration::{ChainInfo, NetworkConfiguration};
-use dcspark_core::{BlockId, BlockNumber, SlotNumber};
-use event::get_parent_id;
+use dcspark_core::BlockId;
 pub use point::*;
 use tokio::sync::{mpsc, oneshot};
 use tokio::time::{interval, Duration, MissedTickBehavior};
-use tracing::{debug, error, info, warn};
+use tracing::{debug, error, info, warn, Instrument};
 
 const TX_PROCESSING_CHANNEL_BOUND: usize = 1000;
```

Comment:
note: this could be configurable probably, what do you think?

Reply:
one more note: cardano source could be placed into its own .rs file, so mod.rs just specifies what should be available externally from the crate. this is just code organization question, not super important, just asking your opinion

Reply:
> note: this could be configurable probably, what do you think?

Well, to be honest this shouldn't matter that much now. In an earlier version the cardano source was also doing the transformation to the bridge IR, so this was meant to stop parsing/deserializing/transforming the blocks if the user was not pulling fast enough. Currently we still deserialize the block here, so it still puts a limit to that, but it should be less of an issue.

I'm fine with making it configurable, although I'm not really sure how worth it is to tweak it. It shouldn't put a bound on memory because AFAIK cardano-net uses an unbounded channel internally, so this only puts a stop on parsing the blocks.

Reply:
well, I think it's always good to have sizes / limits as a parameter, so when you really need it you just change the configuration, not e.g. modify the library

Reply:
#15